src/utils/CommonSchema.ts

```diff
@@ -109,5 +110,6 @@ export const commonSchema = Joi.object().keys({
   createdById: Joi.string().optional(),
   lastUpdatedAt: Joi.string().optional().allow(null),
   lastUpdatedByName: Joi.string().optional(),
-  lastUpdatedById: Joi.string().optional()
+  lastUpdatedById: Joi.string().optional(),
+  statusCode: Joi.string().valid(...STATUS_CODES)
```

Comment:
By default the fields are optional, I think statusCode should be required.

Reply:
statusCode cannot be changed to required as it will cause a lot of unit tests to fail.